tensorflow/core/public/version.h

```diff
@@ -116,5 +116,7 @@ extern const char* tf_compiler_version();
 // The git commit designator when tensorflow was built
 // If no git repository, this will be "internal".
 extern const char* tf_git_version();
+// Value of the _GLIBCXX_USE_CXX11_ABI flag, or -1 if it's not set.
+extern const int tf_cxx11_abi();
```

Comment:
I would rename everything to `_flag`: `tf_cxx11_abi_flag`, `tf.sysconfig.CXX11_ABI_FLAG`